app/static/financial_tool/cost_assumptions.csv

```diff
@@ -0,0 +1,28 @@
+Description;Category;Qty;USD/Unit;Unit;Growth rate;Target
```

Comment:
looks like an excel "csv" which should be called "scsv" for "semi-colon separated values" ;)